### PR TITLE
Fix a regression.

### PR DESCRIPTION
**Pull request by AlekseyTs**
Clloses https://github.com/dotnet/roslyn-analyzers/issues/4839.

### CODE REVIEW

File: src/Compilers/CSharp/Portable/Binder/LockOrUsingBinder.cs [hunk 1]
```diff
@@ -80,15 +80,15 @@ protected BoundExpression BindTargetExpression(BindingDiagnosticBag diagnostics,
             if (_lazyExpressionAndDiagnostics == null)
             {
                 // Filter out method group in conversion.
-                var expressionDiagnostics = BindingDiagnosticBag.GetInstance(diagnostics);
```
Comment (jaredpar) [inline src/Compilers/CSharp/Portable/Binder/LockOrUsingBinder.cs:83]: Is the `diagnostics` parameter used anymore in this method?

Comment (AlekseyTs) [reply src/Compilers/CSharp/Portable/Binder/LockOrUsingBinder.cs:83]: > Is the diagnostics parameter used anymore in this method? 

Yes, eventually ```expressionDiagnostics``` is merged into it.

---
In reply to: [574974823](https://github.com/dotnet/roslyn/pull/51188#discussion_r574974823) [](ancestors = 574974823)

**Comment by JoeRobich** [comment]
PR validation insertion: https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/304158

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review a small fix for a regression blocking an integration of a feature branch.

**Review by cston — APPROVED** [review]

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review a small fix for a regression blocking an integration of a feature branch, need a second sign-off.

File: src/Compilers/CSharp/Portable/Binder/LockOrUsingBinder.cs (hunk 1 above)
Comment (jaredpar) [reply src/Compilers/CSharp/Portable/Binder/LockOrUsingBinder.cs:83]: Doh, GitHub hid the end of the method and I missed it 😦

**Review by jaredpar — APPROVED** [review]

**Pull request merged**